CoreScripting/src/au/gov/asd/tac/constellation/views/scripting/utilities/ScriptingUtilities.java

```diff
@@ -101,7 +100,6 @@ public SGraph copyGraph(final SGraph graph) {
     public void executePlugin(final SGraph graph, final String pluginName) {
         final Plugin plugin = PluginRegistry.get(pluginName);
         final PluginParameters parameters = new PluginParameters();
-        parameters.appendParameters(GlobalParameters.getParameters(null));
```

Comment:
What was the reason behind removing this line?

Reply:
It was unnecessary to add all global parameters to plugins (plugins specify if they need global parameters themselves), and this removes a dependency on the Data Access View module (which makes things more modular).